### PR TITLE
Footer v2

### PR DESCRIPTION
**Pull request by emrerdem1**
Hi Louis! We decided to change the design of footer.

Please let me know if anything looks odd.

Regards.

### CODE REVIEW

File: src/components/layout/footer/Footer.jsx [hunk 1]
```diff
@@ -6,41 +6,56 @@ import {
   faGoogle,
   faGithub,
 } from "@fortawesome/free-brands-svg-icons";
+import Row from "react-bootstrap/Row";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
```
Comment (MiraAr) [inline src/components/layout/footer/Footer.jsx:11]: can you import { Row, Col , Container } from "react-bootstrap"

**Comment by emrerdem1** [comment]
Yes, I can! It's just React-Bootstrap was favouring this way more even
though I'm not sure of the reason.

**Comment by emrerdem1** [comment]
PTAL

**Comment by emrerdem1** [comment]
PTAL

**Review by louisrli — APPROVED** [review]

**Review by MiraAr — APPROVED** [review]

**Pull request merged**